newIDE/app/src/MainFrame/index.js

```diff
@@ -2682,6 +2681,8 @@ const MainFrame = (props: Props) => {
           currentProject: currentProject,
           editorTabs: editorTabs,
         });
+
+        setIsProjectClosedSoAvoidReloadingExtensions(false);
```

Comment:
should you move this in the finally clause?
So that you can remove it from the catch as well

Reply:
Actually, they are not setting the same value. The one in the catch is probably useless, but I think it's safer this way.

Reply:
oh yes, missed that